backend/app.py

```diff
@@ -1 +1,6 @@
-from jina import Flow
+from jina.types.document.generators import from_csv
+from jina import DocumentArray
+
+proteins = DocumentArray(
+    from_csv(open("../data/Train_HHblits_1column.csv"), {"Protein sequences": "text"})
```

Comment:
Based on the documentation, the file path should suffice and probably `open` is unnecessary. Not sure though.

Reply:
If I just pass it a string then it creates a document array where each document is a letter in the string for some reason. Whatever you pass in gets passed to DictReader which I believe needs an open file https://docs.python.org/3/library/csv.html